pkg/lib/machine-add.html

```diff
@@ -1,12 +1,14 @@
 <div class="modal-header">
     <button class="close" data-dismiss="modal"
         aria-hidden="true">&times;</button>
-    <h4 class="modal-title" translate="yes">Add a new host</h4>
+    <h4 class="modal-title" translate="yes">Add new host</h4>
 </div>
 
 <div class="modal-body">
+    <p>Specify the host and the login user account for the host that you want to add.<br/>
+    For the host, either specify the hostname, IP address, an alias name or a unique resource identifier for the SSH destination.</p><br/>
```

Comment:
This is a bit awkward for translations. The `<br/>` is part of the text to be translated, and there's extra line breaks and spaces. Also, these are missing `translate` HTML attributes, so that they get marked for translation.

I suggest splitting the sentences and wrapping them into two `<span>`s, so that the halves get translated separately, without any surrounding whitespace or HTML tags.